docs/DeveloperGuide.md

```diff
@@ -1,34 +1,853 @@
+hours spent in the respective academic weeks.
+
+#### Current Implementation
+
+The feature will implement the same checks as the add module feature as shown earlier.
+
+If the module code is valid and exists the add time feature will proceed to add time from the module
+as indicated by the user.
+
+Given below is an example usage scenario and how the add time command behaves at each step.
+
+Step.1 The user launches the application and does all the required steps to add a module with module code `CS2113T`
+to a list of modules. This is done by inputting the command `addMod CS2113T`.
+
+Step.2 The user now types in a command: `addTime CS2113T 2 4`. This calls the add time command from `Parser`.
+`Parser` then calls the `addTime()` method in `ModuleList` and passes `CS2113T`,`2` and `4` as parameters into the
+method. 
+> **_NOTE:_**  The first number is the hours spent in that week. The second number is the academic week that is being
+>updated.
+>
+
+Step.3 Within the `addTime()` function, it will first check if the module code is valid by calling
+`checkIfModuleValid()` function. If the module code is valid, `checkIfModuleValid()` function will return true.
+
+Step.4 Within the `addTime()` function, it will then check if the module exists in the module list storage by calling
+`checkIfModuleExist()` function. If the module does exist, the `checkIfModuleExist()` function will return true.
+
+Step.5 A time of `2` hours in academic week `4` will now be added to the actual workload of the `CS2113T` module. 
+
+Step.6 The system will print the string `2 hours have been added to CS2113T`
+
+Step.7 The actual workload is updated in storage.
+
+The following sequence diagram illustrates what happens when a user executes `addTime CS2113T 2 4`.
+
+[insert diagram here]
 
-## Product scope
-### Target user profile
+#### Design Considerations
 
-{Describe the target user profile}
+* **Alternative 1 (current choice)**: adding time with academic weeks associated
+    * Pros: Easy to manage for user as the user only needs to focus on the current academic week to track his workload
+    * Cons: User might prefer to manage his time spent in totality
+* **Alternative 2**: adding time in totality 
+    * Pros: Able to cumulatively add time and manage workload based on overall total time spent.
+    * Cons: May not be useful to know total time spent as workload may be better managed weekly.
 
-### Value proposition
 
-{Describe the value proposition: what problem does it solve?}
+### 4.4 Minus Time <a name="4.4"></a>
+The minus time feature edits the actual workload for a specific module as indicated by the user. 
+This is done by removing time spent on the module. The actual workload is broken down into academic weeks and
+hours spent in the respective academic weeks. This feature is the opposite of the add time feature. 
 
-## User Stories
+#### Current Implementation
```

Comment:
Could this section be formatted better?

![image](https://user-images.githubusercontent.com/61812576/97657213-42a4fe00-1aa4-11eb-8007-285c853786ad.png)